src/scores/ais/ais.ts

```diff
@@ -0,0 +1,19 @@
+import { AIS_INPUTS, AIS_OUTPUT, AIS_INTERPRETATION_TABLE } from './definition'
+import { ScoreType } from '../../types'
+import { sum } from 'lodash'
+
+export const ais: ScoreType<typeof AIS_INPUTS, typeof AIS_OUTPUT> = {
+  name: 'Acceptance of Illness Scale (AIS)',
+  readmeLocation: __dirname,
+  inputSchema: AIS_INPUTS,
+  outputSchema: AIS_OUTPUT,
+  calculate: ({ data }) => {
+    const validValues = Object.values(data).filter(value => value !== undefined)
+    const total_score = sum(validValues)
+
+    return {
+      AIS_SCORE: total_score,
+      AIS_INTERPRETATION: AIS_INTERPRETATION_TABLE[total_score.toString()],
```

Comment:
I find this a bit tricky.
- When I answer no questions, the total result will be `0`, which is confusing given the minimum score of the instrument is considered `8`
- Same for when I answer e.g. only 1 question, then the max total score is `5`, which is below `8`

Given this is an 8-item instrument (very short), I'd suggest making the inputs required. If not all 8 items are submitted, the score returns `null`

This happens automatically if you remove the `.optional()` from the zod inputs shape.